velox/docs/develop/spilling.rst

```diff
@@ -407,7 +449,7 @@ level, *M* = 1*GB*, *N* = 3:
      - 21
      - 2 PB
 
-Note that we can set a configurable limit (`kMaxSpillLevel <https://github.com/facebookincubator/velox/blob/main/velox/core/QueryConfig.h#L150>`_) on the max spilling
+Note that we can set a configurable limit (kMaxSpillLevel) on the max spilling
```

Comment:
For production deployments, we recommend setting a limit for the max spilling level using "max-spill-level" configuration property.